spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -166,6 +183,10 @@ def assert_specific_container_service
       :restart_policy => "Always",
       :dns_policy     => "ClusterFirst"
     )
+
+    @containersrv.labels.count.should == 2
```

Comment:
I'd move them to the beginning of the method next to testing attributes, before testing relationships, just for readability.